example/nf_channels_params_template.json

```diff
@@ -0,0 +1,23 @@
+{
+"container_fastqc" : "/path/to/singularity/image/fastqc.sif",
+"container_multiqc" : "/path/to/singularity/image/multiqc.sif",
+"container_trimmomatic" : "/path/to/singularity/image/trimmomatic.sif",
+"container_samtools" : "/path/to/singularity/image/samtools.sif",
+"container_gatk" : "/path/to/singularity/image/gatk.sif",
+"container_bwa" : "/path/to/singularity/image/bwa.sif",
+"container_snpeff" : "/path/to/singularity/image/snpeff.sif",
+"container_bcftools" : "/path/to/singularity/image/bcftools.sif",
+"filesuffix" : "bam or cram",
+"bucket_dir" : "/path/to/bam or cram/files",
+"all_read_pairs" : "/path/to/fastq/files/*_R{1,2}*.fastq.gz",
```

Comment:
Why are the FASTQs included?

Reply:
We were using only 1 params file for both alignment and somatic variant calling, so variables used in the alignment are in there too.